### ISSUE TITLE
Why is creating another app container bad?

### ISSUE THREAD

**Issue by vikramkh** [react-navigation/react-navigation#6020]
When you create another app container, you get a warning that this is not good. What are the possible things  (in laymans terms ) that could happen?

**Comment by matthamil** [comment]
IIRC, under the hood the appcontainer is using [react-native-screens](https://github.com/kmagiera/react-native-screens#screencontainer)'s `<ScreenContainer />`. Check out their docs for info about what it does and how it's used for navigation libraries.
